examples/logical_enclosures.go

```diff
@@ -2,45 +2,73 @@ package main
 
 import (
 	"fmt"
+	"sort"
+	"strings"
+
 	"github.com/HewlettPackard/oneview-golang/ov"
 	"github.com/HewlettPackard/oneview-golang/utils"
-	"os"
-	"strconv"
-	"strings"
 )
 
 func main() {
+
+	config, config_err := ov.LoadConfigFile("config.json")
+	if config_err != nil {
+		fmt.Println(config_err)
+	}
 	var (
 		ClientOV            *ov.OVClient
 		logical_enclosure   = "TestLE"
 		logical_enclosure_1 = "TestLE-Renamed"
 		scope_name          = "Auto-Scope"
-		eg_name             = "EG"
+		eg_name             = config.EgName
 		firmware_baseline   = utils.NewNstring("/rest/firmware-drivers/Synergy_Custom_SPP_2021_02_01_Z7550-97110")
```

Comment:
should we pass this in config?

Reply:
EG name is picked up from config as enclosure group will create it

Reply:
Will remove the firmware baseline